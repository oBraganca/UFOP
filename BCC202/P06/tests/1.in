1 5
1 4
1 6
1 1
5
2 4
2 1
5
3
4 5
-1