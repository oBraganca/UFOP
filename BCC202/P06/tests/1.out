1 4 5 6 
5 6 
Nao Pertence
